udata/core/reuse/models.py

```diff
@@ -113,7 +114,7 @@ def is_visible(self):
 
     @property
     def is_hidden(self):
-        return len(self.datasets) == 0 or self.private or self.deleted
+        return len(self.datasets) == 0 or self.private or self.deleted or self.archived
```

Comment:
Same here, I would not modify it before having discussed product strategy towards archived reuses visibility.

Reply:
Indeed, fixed!